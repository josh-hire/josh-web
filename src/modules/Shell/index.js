import PropTypes from 'prop-types';

export default function Shell({ children }) {
  return children;
}

Shell.propTypes = {
  children: PropTypes.oneOfType([PropTypes.arrayOf(PropTypes.node), PropTypes.node]).isRequired,
};
