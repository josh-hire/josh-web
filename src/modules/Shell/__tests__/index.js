import Shell from '../';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

describe('src/modules/Shell', ()=>{
  test('Shell',()=>{
    const { asFragment } = render(<Shell><div /></Shell>);
    expect(asFragment()).toMatchSnapshot();
  });
});
