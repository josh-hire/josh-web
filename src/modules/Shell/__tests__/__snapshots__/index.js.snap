// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/modules/Shell Shell 1`] = `
<DocumentFragment>
  <div />
</DocumentFragment>
`;
